spec/components/metadata_list_component_spec.rb

```diff
@@ -9,19 +9,11 @@
 
   let(:viewer) { instance_double(Embed::Viewer::Media, purl_object:) }
   let(:purl_object) do
-    instance_double(Embed::Purl,
-                    use_and_reproduction: 'The materials are open for research use',
-                    copyright: 'The Board of Trustees of the Leland Stanford Junior University',
-                    license: 'cc-0',
-                    purl_url: 'https://purl.stanford.edu/gt507vy5436')
+    instance_double(Embed::Purl, purl_url: 'https://purl.stanford.edu/gt507vy5436')
```

Comment:
Will this change trigger network access in the tests?

Reply:
The purl URL was there before. This change is removing code that was already here (extracting rights stuff to its own component spec).